### PR TITLE
Update db dump's name

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: src/tools/db-dump/DbDumper.cpp [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 #include "common/fs/FileUtils.h"
 #include "common/time/Duration.h"
-#include "tools/dbDump/DbDumper.h"
```
Comment (panda-sheep) [inline src/tools/db-dump/DbDumper.cpp:9]: Is there any benefit to doing this?

Comment (darionyaphet) [reply src/tools/db-dump/DbDumper.cpp:9]: unified naming

**Review by bright-starry-sky — APPROVED** [review]
LGTM， thanks for your unified the dir name

**Review by liuyu85cn — APPROVED** [review]
LGTM

**Pull request merged**